package.json

```diff
@@ -153,6 +153,7 @@
     "moment": "2.20.1"
   },
   "scripts": {
+    "gettext": "grunt gettext:extract",
```

Comment:
I'd call it "extract-translation-strings" so it's more obvious what it does.

Reply:
I'll just call it like the task: `gettext-extract`. Just because when I run it with npm I usually remember "gettext" and start typing `npm run gett`... and then I just hit Tab